src/referentielijsten/conf/base.py

```diff
@@ -1,382 +1,42 @@
-            "backupCount": 10,
-        },
-        "project": {
-            "level": "DEBUG",
-            "class": "logging.handlers.RotatingFileHandler",
-            "filename": os.path.join(LOGGING_DIR, "referentielijsten.log"),
-            "formatter": "verbose",
-            "maxBytes": 1024 * 1024 * 10,  # 10 MB
-            "backupCount": 10,
-        },
-        "performance": {
-            "level": "INFO",
-            "class": "logging.handlers.RotatingFileHandler",
-            "filename": os.path.join(LOGGING_DIR, "performance.log"),
-            "formatter": "performance",
-            "maxBytes": 1024 * 1024 * 10,  # 10 MB
-            "backupCount": 10,
-        },
-    },
-    "loggers": {
-        "referentielijsten": {
-            "handlers": ["project"] if not LOG_STDOUT else ["console"],
-            "level": "INFO",
-            "propagate": True,
-        },
-        "django.request": {
-            "handlers": ["django"] if not LOG_STDOUT else ["console"],
-            "level": "ERROR",
-            "propagate": True,
-        },
-        "django.template": {
-            "handlers": ["console"],
-            "level": "INFO",
-            "propagate": True,
-        },
-        "mozilla_django_oidc": {
-            "handlers": ["project"] if not LOG_STDOUT else ["console"],
-            "level": "DEBUG",
-        },
-    },
-}
-
-#
-# AUTH settings - user accounts, passwords, backends...
-#
-AUTH_USER_MODEL = "accounts.User"
-
-AUTH_PASSWORD_VALIDATORS = [
-    {
-        "NAME": "django.contrib.auth.password_validation.UserAttributeSimilarityValidator"
-    },
-    {"NAME": "django.contrib.auth.password_validation.MinimumLengthValidator"},
-    {"NAME": "django.contrib.auth.password_validation.CommonPasswordValidator"},
-    {"NAME": "django.contrib.auth.password_validation.NumericPasswordValidator"},
-]
-
-
-# Allow logging in with both username+password and email+password
-AUTHENTICATION_BACKENDS = [
-    "axes.backends.AxesBackend",
-    "referentielijsten.accounts.backends.UserModelEmailBackend",
-    "django.contrib.auth.backends.ModelBackend",
-    "mozilla_django_oidc_db.backends.OIDCAuthenticationBackend",
-]
-
-SESSION_COOKIE_NAME = "referentielijsten_sessionid"
-SESSION_ENGINE = "django.contrib.sessions.backends.cache"
-
-LOGIN_URL = reverse_lazy("admin:login")
-LOGIN_REDIRECT_URL = reverse_lazy("admin:index")
-LOGOUT_REDIRECT_URL = reverse_lazy("admin:index")
-
 #
 # SECURITY settings
 #
-SESSION_COOKIE_SECURE = IS_HTTPS
-SESSION_COOKIE_HTTPONLY = True
-
-CSRF_COOKIE_SECURE = IS_HTTPS
 CSRF_FAILURE_VIEW = "referentielijsten.accounts.views.csrf_failure"
 
-X_FRAME_OPTIONS = "DENY"
-
-#
-# FIXTURES
-#
-
-FIXTURE_DIRS = (os.path.join(DJANGO_PROJECT_DIR, "fixtures"),)
-
 #
 # Custom settings
 #
 PROJECT_NAME = "Referentielijsten"
-ENVIRONMENT = config("ENVIRONMENT", "")
-SHOW_ALERT = True
-ENABLE_ADMIN_NAV_SIDEBAR = config("ENABLE_ADMIN_NAV_SIDEBAR", default=False)
```

Comment:
@annashamray do you know if these settings are used by any libraries? I couldn't find them anywhere

Reply:
I can't find them either. Let's remove them